lib/tasks/sign_image_tasks.rake

```diff
@@ -1,7 +1,7 @@
 require 'fileutils'
 
 namespace :sign_images do
-  dimensions_regex = Regexp.new(/(\d{1,4})x(\d{1,4})/)
+  DIMENSIONS_REGEXP = Regexp.new(/(\d{1,4})x(\d{1,4})/)
```

Comment:
:elephant: FYI you don't need the `Regexp.new` to create a regex literal - the forward ticks creates a regex literal by itself